Cargo.toml

```diff
@@ -47,6 +47,7 @@ stderrlog = "0.4.1"
 sysconf = ">=0.3.4"
 time = "0.1"
 tiny_http = "0.6"
+rustc-serialize = "0.3.25"
```

Comment:
`.24` just doesn't compile. Has rust compiler errors so updated it to the latest.